advanced-examples/async-init.es6

```diff
@@ -0,0 +1,21 @@
+// jshint ignore: start
```

Comment:
The files was renamed to `delayed-instantiation.js` but the origin got back to the filesystem somehow. Removed it. thanks.